### PR TITLE
update download.md (fixes #500)

### LINKED ISSUE #500: no steps to checksum treehouse.img.gz
### Problem
Under "Setting up your raspberry pi", checksum is provided for downloaded treehouse img.gz but no step is provided to check it

 ### Steps to reproduce the problem
N/A

 ### Screenshots
Screenshot 2020-04-14 at 9 38 13 AM

 ### Proposed solution
Have a step 4 that that completes the download with a shasum check. On mac/linux terminal its shasum <location of .img.gz>

### PR DESCRIPTION
**Pull request by vkabc**
Fixes #500

### Description
Added instructions to do checksum on downloaded img.gz

### Raw.Githack preview link
https://raw.githack.com/jiaxuan-2/jiaxuan-2.github.io/issue500/#!pages/download.md

### CODE REVIEW

**Review by LordJashin32 — APPROVED** [review]
Great job. I think this adds usefulness to the downloads page.

**Review by kjong — APPROVED** [review]

File: pages/download.md [hunk 1]
```diff
@@ -15,6 +15,11 @@ We recommend **[Etcher](https://www.balena.io/etcher/)** to write our images to
   * **[![treehouses](https://avatars1.githubusercontent.com/u/33208073?size=25)treehouses-122.img.gz](http://download.treehouses.io/treehouse-122.img.gz)**
 * **[![treehouses](https://avatars1.githubusercontent.com/u/33208073?size=25)Past Versions](http://download.treehouses.io/)**
 * **[![treehouses](https://avatars1.githubusercontent.com/u/33208073?size=25)Experimental Versions](http://dev.ole.org/)**
+
+
+To do a checksum of the downloaded img.gz, run `shasum "/path/to/img.gz"` in your terminal.
```
Comment (dogi) [inline pages/download.md:20]: @jiaxuan-2 can you transform this into a `Note`

Comment (dogi) [reply pages/download.md:20]: ... forgot notes in mdwiki are special

Comment (vkabc) [reply pages/download.md:20]: Do I resolve the conflict and do I still need to convert this to a note?

Comment (dogi) [reply pages/download.md:20]: @jiaxuan-2 I resolved the conflict for you

Comment (vkabc) [reply pages/download.md:20]: Thank you.

**Review by dogi — APPROVED** [review]

**Pull request merged**